modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -806,6 +804,26 @@ def get_objects_from_partitions(cls, partitions):
         """
         return [partition.get() for partition in partitions]
 
+    @classmethod
+    def wait_partitions(cls, partitions):
+        """
+        Wait on the objects wrapped by `partitions` in parallel, without materializing them.
```

Comment:
```suggestion
        Wait on the objects wrapped by `partitions`, without materializing them.
```
as this specific implementation is _not_ waiting in parallel